src/components/text/index.ts

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-export type { EuiTextProps } from './text';
+export type { EuiTextProps, TextSize } from './text';
```

Comment:
We should not be exporting this type as a top-level prop - use `EuiTextProps['size']` instead
```suggestion
export type { EuiTextProps } from './text';
```